packages/ckeditor5-engine/tests/view/downcastwriter/writer.js

```diff
@@ -122,6 +122,17 @@ describe( 'DowncastWriter', () => {
 			expect( element.id ).to.equal( 'bar' );
 			assertElementAttributes( element, attributes );
 		} );
+
+		it( 'should not treat priority 0 as no priority set', () => {
```

Comment:
```suggestion
		it( 'should pass priority 0', () => {
```

You use negation twice in this sentence. You can simplify it down to something like above.